main.go

```diff
@@ -25,18 +25,18 @@ var (
 	marathonUri = flag.String(
 		"marathon.uri", "http://marathon.mesos:8080",
 		"URI of Marathon")
+	marathonUserName = flag.String(
+		"marathon.username", "", "marathon username")
+	marathonPassword = flag.String(
+		"marathon.password", "", "marathon password")
 )
 
-func marathonConnect(uri *url.URL) error {
+func marathonConnect() error {
 	config := marathon.NewDefaultConfig()
-	config.URL = uri.String()
+	config.URL = *marathonUri
 
-	if uri.User != nil {
-		if passwd, ok := uri.User.Password(); ok {
-			config.HTTPBasicPassword = passwd
-			config.HTTPBasicAuthUser = uri.User.Username()
-		}
-	}
+	config.HTTPBasicAuthUser = *marathonUserName
```

Comment:
So, user is not able to specify its marathon user and password in the URL anymore? It would be ncie to be able to do both for retro compatibility